versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -63,6 +64,7 @@ class CaffeineCacheBackend implements CacheBackend {
 
     Caffeine<CacheKeyValue, CacheKeyValue> cacheBuilder =
         Caffeine.newBuilder()
+            .scheduler(Scheduler.systemScheduler())
             .maximumWeight(config.capacityMb() * 1024L * 1024L)
             .weigher(this::weigher)
             .expireAfter(
```

Comment:
ditto?

Reply:
We need `expireAfterCreate` and ``expireAfterUpdate` here.
And would require a bigger change, as all supplying function are defined to provide the expiration timestamp.

Reply:
Oops, so `Expiry.writing`?